playbooks/bastion/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
I'd suggest removing this file as it's an `optional` setting

Reply:
Even if you included it, maybe just leave it commented out with a comment about it being option - if so, please add it to the `bastion.yml` file rather than introducing an `all.yml`